db/schema.rb

```diff
@@ -84,6 +84,8 @@
     t.string   "confirmation_token"
     t.datetime "confirmed_at"
     t.datetime "confirmation_sent_at"
+    t.string   "name"
```

Comment:
Can we instead call this `first_name` or something similar to conform to the pattern from `last_name`?